### DISCUSSION TITLE
The radio component id processing method.

### DISCUSSION THREAD

**Discussion by chenxch** [element-plus/element-plus#8653] [RFC]
## background reasons
   Since the radio component is wrapped with a label tag, the id attribute is written on the el-radio tag. This id attribute will be on the label tag by default, so that the external user's own label tag cannot respond to the corresponding double-click after adding the for attribute. The id of the radio.
  Later, it was fixed in #8476 . The fix was to keep style, class and listieners in the label tag, and all other attrs were given to the input tag.

## currently existing problems
#8641 Custom attributes are extracted onto the input.

In fact, the essence of this problem should be to bind the id to the input. There are currently several options.

**Comment by Giwayume** [comment]
If the user wants to use their own label, the label that is generated by `el-radio` is pointless and should be removed.

Problem is, much of the logic in `el-radio` relies on the browser automatically checking the radio button when the label is clicked. That needs to be disassociated so no label is generated by `el-radio`. Otherwise there is no point in letting the user attach their own label, because the browser will be confused as to which radio to use due to multiple labels being attached.

**Reply by chenxch** [reply]
Suddenly, it may not be necessary to make the component difficult to maintain or use in order to be compatible with the native thing. If the user needs to implement this function, he can add a click event to the label to handle it.
